docs/resources/metric_metadata.md

```diff
@@ -36,7 +36,7 @@ resource "datadog_metric_metadata" "request_time" {
 - `per_unit` (String) Per unit of the metric such as `second` in `bytes per second`.
 - `short_name` (String) A short name of the metric.
 - `statsd_interval` (Number) If applicable, statsd flush interval in seconds for the metric.
-- `type` (String) Metric type such as `gauge` or `rate`.
+- `type` (String) Metric type such as `count`, `gauge`, or `rate`. Updating a metric of type `distribution` is not supported. If you would like to see the `distribution` type returned, please contact Datadog support.
```

Comment:
```suggestion
- `type` (String) Metric type such as `count`, `gauge`, or `rate`. Updating a metric of type `distribution` is not supported. If you would like to see the `distribution` type returned, contact Datadog support.
```

Reply:
It might be helpful to link "Datadog support" to https://docs.datadoghq.com/help/.

Also, since you mention contacting support, could you also add an entry to this doc so Support knows how to handle the request? https://datadoghq.atlassian.net/wiki/spaces/TS/pages/328404964/The+Public+Docs+say+Contact+Support